airflow-ctl/src/airflowctl/api/operations.py

```diff
@@ -243,8 +243,11 @@ def create_event(
     ) -> AssetEventResponse | ServerResponseError:
         """Create an asset event."""
         try:
+            # Ensure extra is an object; DB/UI choke on null
```

Comment:
There is no `choke` on

```suggestion
            # Ensure extra is initialised before sent to API
```